smart-send-logistics/build/smart-send-block-frontend.js

```diff
@@ -0,0 +1,667 @@
+/******/ (() => { // webpackBootstrap
+/******/ 	"use strict";
+/******/ 	var __webpack_modules__ = ({
+
+/***/ "./src/js/smart-send-block/block.js":
+/*!******************************************!*\
+  !*** ./src/js/smart-send-block/block.js ***!
```

Comment:
Where is this file?

Reply:
The code above is part of a compiled file, and this line indicates the source from which the current code has been compiled. I have removed it as you mentioned it previously.

Reply:
But if you have removed the file, yet it is still referenced here, then that would indicate that the build process has not be done?